### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute (part 3)

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only build validation fixes and does not change other content.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: @MonicaRush
CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514

### CODE REVIEW

File: docs/shared/office-shared.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ Find how-to content, sample code, SDK and API documentation, VBA references, tra
...
   
-- [Platform overview](https://docs.microsoft.com/previous-versions/office/office-365-api/)
+- [Platform overview](/previous-versions/office/office-365-api/)
   
 ### Office Add-ins
   
-- [Get started](https://docs.microsoft.com/office/dev/add-ins/)
+- [Get started](/office/dev/add-ins/)
 
 - [Platform overview](https://docs.microsoft.com/office/dev/add-ins/overview/office-add-ins)
```
Comment (MeeraDi) [inline docs/shared/office-shared.md:38]: Platform overview link was skipped

**Comment by MeeraDi** [comment]
#docutune-wait

File: docs/shared/office-shared.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/shared/office-shared.md]: Corrected

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@AlexJerabek @dstrome

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Pull request merged**